app/kumactl/cmd/delete/delete_meshes.go

```diff
@@ -0,0 +1,37 @@
+package delete
+
+import (
+	"context"
+	"fmt"
+	"github.com/Kong/kuma/pkg/core/resources/apis/mesh"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func newDeleteMeshCmd(pctx *deleteContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "mesh",
+		Short: "Delete Mesh",
+		Long:  `Delete Mesh.`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			rs, err := pctx.CurrentResourceStore()
+			if err != nil {
+				fmt.Println("Error occurred when getting store")
```

Comment:
Don't print error message in here, just `return err`.

If you want to change error message, use `errors.Wrapf(err, "additional error message")`